configure

```diff
@@ -1545,7 +1543,8 @@ Optional Packages:
   --with-perl             build Perl modules (PL/Perl)
   --with-python           build Python modules (PL/Python)
   --with-java             build Java modules (PL/Java)
-  --with-r             build R modules (PL/R)
+  --with-r                build R modules (PL/R)
```

Comment:
It would be better to use:
```
--with-plperl
--with-plpython
--with-pljava
--with-plr
--with-pgcrypto
```

Also the comment for "with-pgcrypto" should be "build with pgcrypto"